src/shared/api/services/ActivityItemDto.ts

```diff
@@ -717,6 +718,23 @@ type StabilityTrackerConfiguration = {
 
 export type FlankerAnswerSettings = null;
 
+type RequestHealthRecordDataConfiguration = ButtonsConfiguration & {
+  skippableItem?: boolean;
+};
+
+type RequestHealthRecordDataAnswerSettings = {
+  optInOutOptions: [
+    {
+      id: 'opt_in';
+      label: string;
+    },
+    {
+      id: 'opt_out';
+      label: string;
+    },
+  ];
+};
+
```

Comment:
Based on your own suggestion in my PR:

```suggestion
export enum EHRConsent {
  OptIn = 'opt_in',
  OptOut = 'opt_out',
}

export 
type RequestHealthRecordDataAnswerSettings = {
  optInOutOptions: [
    {
      id: EHRConsent.OptIn;
      label: string;
    },
    {
      id: EHRConsent.OptOut;
      label: string;
    },
  ];
};
```

P.S. I called the enum `EHRConsent` because it's a lot shorter than `RequestHealthRecordDataItemConsent` and wanted to keep things concise. 🤓

Reply:
Awesome! Adding this in